[2023-12-14 11:29:46] [info] server.go:616 dpchain/p2p/server.(*Server).setupConn p2pServer等待完成与目标节点(127.0.0.1:20133)的加密handshake与协议handshake!

[2023-12-14 11:29:46] [info] server.go:602 dpchain/p2p/server.(*Server).listenLoop 接收到来自节点 127.0.0.1:61557的p2p连接请求

[2023-12-14 11:29:46] [info] server.go:616 dpchain/p2p/server.(*Server).setupConn p2pServer等待完成与目标节点(127.0.0.1:61557)的加密handshake与协议handshake!

[2023-12-14 11:29:46] [info] server.go:636 dpchain/p2p/server.(*Server).setupConn p2pServer完成与目标节点(127.0.0.1:61557)的加密handshake!

[2023-12-14 11:29:46] [info] server.go:643 dpchain/p2p/server.(*Server).setupConn 下面进行加密handshake的 checkpoint!!! ,127.0.0.1:61557 

[2023-12-14 11:29:46] [info] server.go:636 dpchain/p2p/server.(*Server).setupConn p2pServer完成与目标节点(127.0.0.1:20133)的加密handshake!

[2023-12-14 11:29:46] [info] server.go:643 dpchain/p2p/server.(*Server).setupConn 下面进行加密handshake的 checkpoint!!! ,127.0.0.1:20133 

[2023-12-14 11:29:46] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn inbound conn 79f9062bfd04143f 127.0.0.1:61557 failed checkpoint posthandshake: Connected to self
[2023-12-14 11:29:46] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn dyn dial conn 79f9062bfd04143f 127.0.0.1:20133 failed checkpoint posthandshake: Connected to self
